packages/breakdown-trees/package.json

```diff
@@ -113,7 +120,8 @@
     "@bentley/ui-ninezone": "^2.19.9",
     "@bentley/webgl-compatibility": "^2.19.9",
     "@bentley/webpack-tools": "1.14.1",
-    "@bentley/measure-tools-react": "^0.5.0",
+    "@bentley/measure-tools-react": "0.5.0",
```

Comment:
shouldnt this be using workspace too?

Reply:
and should you really be pinning this down? id suggest copying how core does their peer deps

Reply:
done.